searchlib/src/vespa/searchlib/common/packets.h

```diff
@@ -42,6 +42,12 @@ enum fnet_feature_masks {
                                GDF_RESCLASSNAME |
                                GDF_PROPERTIES |
                                GDF_FLAGS),
+    FNET_QFLAG_SUPPORTED_MASK = (QFLAG_EXTENDED_COVERAGE |
```

Comment:
'SUPPORTED' is misleading since one would expect unsupported variants to be dropped (like with features), not transitively ignored.

Reply:
Also, the interpretation of query flags is done on the application level and not on the protocol level. What if we change the behavior related to a query flag?

Reply:
I agree to the first comment. Perhaps SUPPORTED -> REQUESTED is better.

Reply:
I also think the flag should be owned by fdispatch on an application level rather than down in the protocol.

Reply:
and the filtering should be performed when the incoming qflags is forwarded to outgoing packets.